server/src/main/java/fr/uga/l3miage/tp4/models/CandidateEvaluationGridEntity.java

```diff
@@ -0,0 +1,29 @@
+package fr.uga.l3miage.tp4.models;
+
+
+
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Entity
+@Getter
+@Setter
+
+public class CandidateEvaluationGridEntity {
+    @Id
+    private Long sheetNumber;
+
+    private double grade;
+
+    private LocalDateTime submissionDate;
+
+    @ManyToOne
+    private ExamEntity examinerEntity;
+
+    @ManyToOne
+    private CandidateEntity candidateEntity;
+}
```

Comment:
ici il faut mettre aussi le `@ManyToMany` sur `EvaluationCriteriaEntity` et pas l'inverse
```suggestion
    private CandidateEntity candidateEntity;
    
    @ManyToMany
    private Set<EvaluationCriteriaEntity> evaluationCriteriaEntities;
}
```